client/src/pages/characters.css

```diff
@@ -1,16 +1,32 @@
 .character-list {
   display: grid;
-  gap: 1rem;
+  gap: 1.5rem;
   grid-template-columns: 1fr;
   place-content: center;
 }
 
 /* Interviewee - add styles for the character list layouts */
 /* screen >= `30rem`, `2` per row */
+/* csw: took some research to learn how to get to screen size in css... now I know */
```

Comment:
Awesome!  The goal is to always learn something.  Reach out of our comfort zone.